data/remnant/remnant 2 cognizance.txt

```diff
@@ -1360,11 +1367,9 @@ mission "Remnant: Cognizance 32"
 	description "Chilia has asked you to pick up 58 tons of refined fuel from <planet> by <date>."
 	source "Viminal"
 	destination "Clink"
-	blocked "Chilia and Taely look excited about something, but when you inquire, they shake their heads and say you need to have earned their capital license before they can tell you more."
```

Comment:
This line is intentionally here so that players know that yes, there is a mission here that they can't get right now, and what's missing.

I don't see any reason for not telling them that they're missing out on something here.

Reply:
As it is, the only way for this mission to be blocked is if the player dosn't have the remnant capital license, which will only be the case if they haven't gotten far enough in the Salvage chain, in which case this mission should just not attempt to offer and instead those missions can be offered.

This situation may warrant a more complex way of guiding the player, though.